### ISSUE TITLE
why the value of loss and acc is better when has desyn?

### ISSUE THREAD

**Issue by BigPillow** [ANSSI-FR/ASCAD#10]
Hi,
Does this mean that mean rank have nothing to do with loss and acc in train?
you said that the desynchronization only deal with test_deta, not profiling_data.  i wonder why the value of loss and acc will different..

**Comment by rstrullu** [comment]
There is indeed a relationship between crossentropy, accuracy, and mean rank, but these metrics are not equivalent. During training, if the accuracy is high or the crossentropy is low (above/below the values obtained by a random classifier, i.e. 1/256 for the accuracy and log(256) for the crossentropy), then the mean rank will approach 0. However, from our experience, the converse is not true : it happened that our classifier has overfitted during training (high crossentropy and low accuracy), but the mean rank still tends to 0 when the number of traces increases and the overall side-channel attack succeeds. This means that in the case of SCA, we can not rely exclusively on the value of the crossentropy and accuracy to assess the success of the attack.
